src/maintainer/knowledge_base.rst

```diff
@@ -665,6 +665,17 @@ BLAS ``3.8.0`` API.  This means that, at install time, the user can select what
 they like without any knowledge of the version of the BLAS implementation needed.
 
 
+.. _knowledge:mpl:
+
+Matplotlib
+----------
+
+``matplotlib`` on ``conda-forge`` comes in two parts. The core library is in ``matplotlib-base``. The
+actual ``matplotlib`` package is this core library plus ``pyqt``. Most, if not all, packages that have
+dependence at runtime on ``matplotlib`` should list this dependence as ``matplotlib-base`` unless they 
+explicitly need ``pyqt``. ``pyqt`` is a rather large package, so not requiring it is better for users.
```

Comment:
```suggestion
explicitly need ``pyqt``. ``pyqt`` is a rather large package, so not requiring it indirectly is better for users.
Rationale is that a user installing `matplotlib` explicitly would get the batteries included version, but
it should not be installed indirectly.
```

Reply:
matplotlib needs double backticks

Reply:
Sure! Feel free to make any changes at all!

Reply:
I took the liberty to edit the comment. Feel free to commit.

Reply:
@beckermr, can you push this? github says I don't have permission

Reply:
The `Allow edits from from maintainers` button in PRs allows you:
* to push to the branch
* edit in the PR web interface

it doesn't allow you to:
 * accept suggestions
 * edit using the non-PR web interface by browsing to the repository and clicking edit

Reply:
Ahhhh sorry. Let me accept.

Reply:
Done.